ansibullbot/triagers/plugins/shipit.py

```diff
@@ -261,6 +261,14 @@ def get_shipit_facts(issuewrapper, inmeta, module_indexer, core_team=[], botname
     ]
 
     files = [f for f in iw.files if not f.startswith(u'changelogs/fragments/')]
+
+    # https://github.com/ansible/ansibullbot/issues/1238
+    meta[u'component_matches'] = [
+        x for x in meta.get(u'component_matches', [])
+        if not x[u'repo_filename'].startswith(u'test/sanity')
+    ]
+    files = [f for f in files if not (f.startswith(u'test/sanity') and f.endswith(u'ignore.txt'))]
```

Comment:
What makes this only work for deletions to these files and not additions/changes?

Reply:
Evaluating additions vs deletions is done in the automerge inspection. This function wasn't well designed for the increasing scope so i'm thinking about a better way to refactor it all.